### PR TITLE
Feature/toast hook

### PR DESCRIPTION
**Pull request by jordanlesich**
## GitHub Issue

https://github.com/HausDAO/daohaus-monorepo/issues/230

## Changes

See issue description. 

Current limitations: 
- Does not handle multiple toasts atm. We need to see how animation shakes out first. 

## Packages Added

NA

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

### CODE REVIEW

File: libs/ui/src/components/molecules/Toast/Toast.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ type CustomToastProps = ToastProps &
     toastLinks?: ToastLinksProps;
   };
 
-type ToastLinksProps = {
+export type ToastLinksProps = {
```
Comment (0xRowdy) [inline libs/ui/src/components/molecules/Toast/Toast.tsx:41]:
```suggestion
type ToastLinksProps = {
```
Don't think we're using ToastLinkProps anywhere outside this file.

File: libs/ui/src/hooks/useToast.tsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    title: string;
+    description?: string;
+  }): void => {
+    setToast({
+      title,
+      description,
+      error: true,
+      iconType: 'error',
+    });
+  };
```
Comment (0xRowdy) [inline libs/ui/src/hooks/useToast.tsx:20]:
```suggestion
  };
  
```
Line break?

File: libs/ui/src/hooks/useToast.tsx [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    title: string;
+    description?: string;
+  }): void => {
+    setToast({
+      title,
+      description,
+      success: true,
+      iconType: 'success',
+    });
+  };
```
Comment (0xRowdy) [inline libs/ui/src/hooks/useToast.tsx:34]:
```suggestion
  };
  
```

**Review by 0xRowdy — APPROVED** [review]

File: libs/ui/src/components/molecules/Toast/Toast.tsx (hunk 1 above)
Comment (jordanlesich) [reply libs/ui/src/components/molecules/Toast/Toast.tsx:41]: I'm using it in the context now. I can refactor it somewhere higher up.

Comment (0xRowdy) [reply libs/ui/src/components/molecules/Toast/Toast.tsx:41]: It looked like CustomToastProps was used in context. ToastLinkPros is a part of CustomToastProps.

**Pull request merged**